L1Trigger/L1TCaloLayer1/src/L1TCaloLayer1FetchLUTs.cc

```diff
@@ -235,12 +254,12 @@ bool L1TCaloLayer1FetchLUTs(const edm::EventSetup& iSetup,
         auto index = std::distance(hfScalePhiBins.begin(),pos);
         if (index<18) {
           caloEta*=-1;
-          iPhi = index*4+1;
+          iPhi = index*4-1;
         }
         else {
-          iPhi = (index-18)*4+1;
+          iPhi = (index-18)*4-1;
         }
-        if (abs(caloEta) == 41 && iPhi<3) iPhi = 3;
```

Comment:
The reason for this was: https://github.com/nsmith-/cmssw/blob/57ac135f3ea01c077ba566ca6a7129eec399e966/L1Trigger/L1TCaloLayer1/src/L1TCaloLayer1FetchLUTs.cc#L106.

Not sure if still accurate.